flytekit/annotated/base_task.py

```diff
@@ -160,7 +164,14 @@ def __call__(self, *args, **kwargs):
             ctx.execution_state is not None and ctx.execution_state.mode == ExecutionState.Mode.LOCAL_WORKFLOW_EXECUTION
         ):
             if ctx.execution_state.branch_eval_mode == BranchEvalMode.BRANCH_SKIPPED:
-                return
+                if self.python_interface.custom_interface_name:
+                    variables = [k for k in self.python_interface.outputs.keys()]
+                    output_tuple = collections.namedtuple(self.python_interface.custom_interface_name, variables)
+                    nones = [None for _ in self.python_interface.outputs.keys()]
+                    return output_tuple(*nones)
+                else:
```

Comment:
Will it always have a Python Interface at this point? and the question is whether the named tuple should have a name or not?

Reply:
I think we'll eventually merge PythonTask and Task, so things should make more sense then, but that might not be the case.  Currently yes, it will always have a python interface at this point because the only Tasks are PythonTasks currently but the code technically allows a non-PythonTask.